.github/workflows/ci.yml

```diff
@@ -525,7 +468,7 @@ jobs:
   wait-for-ci-images:
     timeout-minutes: 120
     name: "Wait for CI images"
-    runs-on: "${{needs.build-info.outputs.runs-on}}"
```

Comment:
I thinl we will save quite a lot of "self-hosted" time for committer builds this way.